Projects/UOContent/Items/Lights/Lantern.cs

```diff
@@ -6,10 +6,12 @@ namespace Server.Items;
 [SerializationGenerator(0, false)]
 public partial class Lantern : BaseEquipableLight
 {
+    protected static TimeSpan FullDuration = TimeSpan.FromMinutes(20);
```

Comment:
I didn't realize it was used outside of Lantern.

```suggestion
    public static TimeSpan FullDuration = TimeSpan.FromMinutes(20);
```